packages/rolldown/package.json

```diff
@@ -103,15 +103,15 @@
     "dtsHeader": "type MaybePromise<T> = T | Promise<T>\ntype Nullable<T> = T | null | undefined\ntype VoidNullable<T = void> = T | null | undefined | void\nexport type BindingStringOrRegex = string | RegExp\n\n"
   },
   "dependencies": {
-    "@oxc-project/types": "0.48.2",
+    "@oxc-project/types": "0.51.0",
```

Comment:
I'll add a `just` command for updating oxc next time, we need to keep all the versions in sync.